manifests/cf-manifest/deployments/050-logsearch.yml

```diff
@@ -176,9 +180,7 @@ jobs:
       templates:
       - shards-and-replicas: "{ \"template\" : \"*\", \"order\" : 99, \"settings\" : { \"number_of_shards\" : 1, \"number_of_replicas\" : 0 } }"
       - index_template: /var/vcap/packages/logsearch-config/default-mappings.json
-    logstash_parser:
```

Comment:
I think you added this change by accident. You don't need to remove this to disable metrics collection.